arbor/affinity.hpp

```diff
@@ -39,11 +39,13 @@ inline
 void set_affinity(int index, int count, affinity_kind kind) {
     // Create the topology and ensure we don't leak it
     auto topology = hwloc_topology_t{};
-    auto guard = util::on_scope_exit([&] { hwloc_topology_destroy(topology); });
+    auto topology_guard = util::on_scope_exit([&] { hwloc_topology_destroy(topology); });
     hwloc(hwloc_topology_init(&topology), "Topo init");
     hwloc(hwloc_topology_load(topology), "Topo load");
     // Fetch our current restrictions and apply them to our topology
-    hwloc_cpuset_t cpus{};
+    hwloc_cpuset_t cpus = hwloc_bitmap_alloc();
```

Comment:
Now I am convinced I messed something up with git ... I have this precise change somewhere. Likely on the system I tested with :D

Reply:
:)